blog/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+
+from blog.models import Post, Commentary
+
+admin.site.unregister(Group)
+admin.site.register(Post)
+admin.site.register(Commentary)
```

Comment:
`Do the filtering and searching in any way you think is logical.`

This is not done